demos/imagenet_classification_demo/classification_results.h

```diff
@@ -0,0 +1,86 @@
+// Copyright (C) 2019 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <string>
+#include <vector>
+#include <iostream>
+#include <utility>
+
+#include <ie_blob.h>
+
+class ClassificationResult {
+public:
+    explicit ClassificationResult(InferenceEngine::Blob::Ptr output_blob,
+                                  size_t batch_size = 1,
+                                  size_t num_of_top = 10,
+                                  std::vector<std::string> image_names = {},
+                                  std::vector<std::string> labels = {}) :
+            _nTop(num_of_top),
+            _outBlob(std::move(output_blob)),
+            _labels(std::move(labels)),
+            _imageNames(std::move(image_names)),
+            _batchSize(batch_size) {
```

Comment:
None of vars is used.
Removing them will eliminate a need in introducing the class.